File: unit-2/sprint-1/day1-1_Activity and LifeCycle/you_/Display Toast Message/app/src/main/java/com/example/toast/MainActivity.java
package com.example.toast;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    Button mBtnFirst;
    Button mBtnSecond;
    Button mBtnThird;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        mBtnFirst = findViewById(R.id.btnFirst);
        mBtnSecond = findViewById(R.id.btnSecond);
        mBtnThird = findViewById(R.id.btnThird);
        mBtnFirst.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(MainActivity.this, "Button One clicked", Toast.LENGTH_SHORT).show();
            }
        });
        mBtnSecond.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(MainActivity.this, "Button second clicked", Toast.LENGTH_SHORT).show();
            }
        });
        mBtnThird.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(MainActivity.this, "Button Third clicked", Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: unit-4/sprint-2/Evaluation/app/src/main/java/com/example/myapplication/MainActivity.kt
package com.example.myapplication

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.LinearLayoutManager
import kotlinx.android.synthetic.main.fragment_add_item.*
import kotlinx.android.synthetic.main.fragment_search_item.*

class MainActivity : AppCompatActivity() {
//    private var tasksList = mutableListOf<Task>()
//    private val itemModel = listOf<Task>()
//    private val dataBaseHandler = DataBaseHandler(this)
//    private lateinit var adapter: ItemAdapter
//    override fun onCreate(savedInstanceState: Bundle?) {
//        super.onCreate(savedInstanceState)
//        setContentView(R.layout.activity_main)
//        initRecyclerView()
//        btnSave.setOnClickListener {
//            addItem()
//        }
//        btnGetItem.setOnClickListener {
//            getData()
//        }
//    }
//
//    private fun getData() {
//        tasksList.clear()
//        tasksList.addAll(dataBaseHandler.getAllItems())
////        liveData.getData(tasksList.size)
//        adapter.notifyDataSetChanged()
//    }
//
//    private fun addItem() {
//        dataBaseHandler?.insertItem(
//            etItemName.text.toString(),
//            etPrice.text.toString().toInt(),
//            etDesc.text.toString())
//    }
//    fun initRecyclerView(){
//        val LayoutManager = LinearLayoutManager(this)
//        adapter = ItemAdapter(itemModel,tasksList)
//        recyclerView.adapter = adapter
//        recyclerView.layoutManager = LayoutManager
//    }

    private lateinit var adaptor: ItemAdapter
    private val addModel = listOf<Task>()
    private var tasksList = mutableListOf<Task>()
    private val dataBaseHandler = DataBaseHandler(this)
//    private val liveData = LiveData()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnSave.setOnClickListener {
            dataBaseHandler?.insertItem(etItemName.text.toString(),
                etPrice.text.toString().toInt(),
                etDesc.text.toString())
        }

        btntoSearch.setOnClickListener {
            tasksList.clear()
            tasksList.addAll(dataBaseHandler.getSearchTask(ettoSearch.text.toString()))
   //         liveData.getData(tasksList.size)
            adaptor.notifyDataSetChanged()

        }


//        liveData.getcountData().observe(this, Observer {
//            tvCount.text = it
//        })
        val LayoutManager = LinearLayoutManager(this)
        adaptor = ItemAdapter(addModel, tasksList)
        RecyclerViewSearch.adapter = adaptor
        RecyclerViewSearch.layoutManager = LayoutManager
    }
}